tests/webapp/api/test_performance_data_api.py

```diff
@@ -375,3 +375,34 @@ def test_filter_data_by_signature(client, test_repository, test_perf_signature,
             assert len(resp.data[signature.signature_hash]) == 1
             assert resp.data[signature.signature_hash][0]['signature_id'] == signature.id
             assert resp.data[signature.signature_hash][0]['value'] == float(i)
+
+
+def test_perf_by_revision(client, test_perf_signature, test_perf_data):
+
+    query_params1 = '?repository={}&framework={}&interval=172800&no_subtests=true&revision={}'.format(
+        test_perf_signature.repository.name, test_perf_signature.framework_id, test_perf_data[0].push.revision)
+
+    query_params2 = '?repository={}&framework={}&interval=172800&no_subtests=true&startday=2018-07-01T23%3A28%3A29&endday=2018-12-31T23%3A28%3A29'.format(
+                    test_perf_signature.repository.name, test_perf_signature.framework_id)
+
+    expected = [{
+        u'id': test_perf_signature.id,
```

Comment:
Bikeshed: I don't the the 'u' prefixes are needed here (I know the unit test output includes that prefix on expected output on failure, but it isn't really required)

Reply:
Ok, I'll see about removing them.